### DISCUSSION TITLE
Add "Remind me about this comment" feature

### DISCUSSION THREAD

**Discussion by tgr** [element-hq/element-meta#798] [Ideas]
I have found Slack's "Remind me about this" feature suprisingly useful. A standard problem with notifications is that you might read a message, decide it's important but you can't deal with it right now, and you want to make sure you don't forget it. This comes up commonly when Element is used for work or otherwise consequential discussions.

Some chat systems have a "mark unread" option (I think Element had it at some point? or maybe some other version of it than web has?) but that tends not to work well, as a read marker that's just a position in a comment feed cannot be meaningfully used for marking specific comments, and a per-comment read marker would be unmanageable in the UI. Instead (or besides, as it does have a not-very-useful "mark unread" option), Slack provides a reminder functionality in the message context menu:

![slack-remindme-1](https://user-images.githubusercontent.com/145412/99142746-56319680-2658-11eb-9969-6e896be48aa9.png)

![slack-remindme-2](https://user-images.githubusercontent.com/145412/99142753-69446680-2658-11eb-91f2-0d7a10272710.png)

And then you get a direct message from the bot with a deep link to the message and a snooze option:

![slack-remindme-3](https://user-images.githubusercontent.com/145412/99143390-351f7480-265d-11eb-84fc-b3086f9c59c6.png)
![slack-remindme-4](https://user-images.githubusercontent.com/145412/99143393-3781ce80-265d-11eb-880f-6c799cc1ec1a.png)

It is great. I would love to see such a feature in Element. I'm not sure how much would be technically "inside" in a strict technical sense (I suppose most of it is a bot and not the Slack client itself, although at least the context menu is part of the client), but for Slack this is something you get by default and the experience feels very smooth and integrated. I think Element and EMS would benefit greatly from similar functionality.

(There are some reminder bots for Matrix, but they are limited to sending you some text you give them manually, and the UI seems to be basically nonexistent.)

**Comment by ShadowJonathan** [comment]
I thought i remembered there was a "star"/"label"/"bookmark" issue around here somewhere too, and i also think this is plenty related to pinned messages and such, should we make a meta-issue that links them all together?

**Reply by yanndinendal** [reply]
It feels to me completely unrelated to pinned messages (which are public and global, and have no notifications).

**Comment by Stvad** [comment]
is anyone aware of a good workaround/bot for this btw? (this is one of the features I really miss from slack)

**Comment by shoqvalue** [comment]
Is there some reason this issue doesn't get more discussion? It's one of the things i miss most in Slack, and every single time I bring in a new user from Slack, they want to know where it is, as it's such a popular thing.  I realized bots "could" do this, but users don't wonder about what might be doable with a little research. They just want the tools they have come to expect in modern chat apps like Slack and Element. I really think simple features like this would greatly enhance the element experience.

**Comment by vitriolix** [comment]
Agree @shoqvalue , this issue covers a really important feature that enables a sensible way of dealing with the flood of messages and losing tasks in the shuffle

**Comment by r1lax** [comment]
That is an extremely useful feature to be able to be reminded about doing something directly in the client.
setup reminder about reading a comment or an issue, especially while multitasking..

Please change the label on that idea: occasional.. it's one of the function I used the most on slack..

I also don't get why this is seen as a notification.. it is not a notification, it is a todo list feature... associated to a timestamp - not at all a notification that you got messaged..

**Reply by r1lax** [reply]
I would broaden the feature and not make it focus on a specific message.
my use case, 
/remind me tomorrow at 9am to speak about xx with yy (that is triggered after reading a specific message)
/remind me tomorrow at 9am to follow up on xx with yy (that is triggered after asking update / or waiting for feedback)
/remind xx at 2pm about the call with YY

this remind feature in slack is very versatile and very useful

**Reply by anarute** [reply]
For the slash command you can use maubot's Reminder: https://github.com/maubot/reminder

**Reply by r1lax** [reply]
> For the slash command you can use maubot's Reminder: https://github.com/maubot/reminder

thank you!

**Comment by Penguin-in-Matrix** [comment]
I am working for a faculty of an university and we are trying to convince all people from our chairs to move from Slack to Element with Matrix. People here already said that they find the notification feature of Slack very useful and asked were they could find such a feature in Element. It is always difficult to convince people changing one working system for another. In the best case the new system has all features of the previous one, plus more features.

It would be great if we here in this thread could convince the development team of Element to implement such a feature. They are doing a great job.

**Comment by r1lax** [comment]
![image](https://github.com/user-attachments/assets/a92a8c7c-34c7-4c01-ab74-8771b1377a63)
Would be awesome to have this included in the client.
as PM it is not an occasional feature, I truly miss this one feature since I transitioned from slack to matrix
